reactions/src/SPT.cpp

```diff
@@ -712,6 +732,26 @@ static double ploopn2_mgdd_pseudo( const PowerSpectrum& P_L, double vars[], doub
     return pow2(r)*2.*P_L(k*r)*(P_L(kargs[0])*p22 + 3.*P_L(k)*p13);
 }
 
+// BILL MOD
+static double ploopn2_mgdd_pseudo_nu( const PowerSpectrum& P_L, double vars[], double k, double r, double x){
+  double kargs[4],kv[3],xv[3], p22,p13;
+  IOW iow;
+        kv[0] = k;
+        kv[1] = k*r;
+        kv[2] = kv[1];
+        xv[0] = -0.99999999; // don't set to -1 otherwise solver encounters singularity
+        xv[1] = x;
+        xv[2] = -x;
+        kargs[0] = sqrt(kv[1]*kv[1]+kv[0]*kv[0]-2.*kv[1]*kv[0]*xv[1]);
+        kargs[2] = sqrt(kv[1]*kv[1]+2.*kv[1]*kv[2]*xv[0]+kv[2]*kv[2]);
+        kargs[1] = sqrt(kv[2]*kv[2]+2.*kv[2]*kv[0]*xv[2]+kv[0]*kv[0]);
+        kargs[3] = sqrt(kv[0]*kv[0]+2.*kv[0]*kv[1]*xv[1]+kv[1]*kv[1]);
+        iow.initn2_pseudo(vars[0],kv,xv,kargs,vars[1],vars[2],vars[3],vars[4]);
+        p22 = pow2(F2_nk);
+        p13 = F1_nk*F3_nk;
+
+    return pow2(r)*2.*(P_L(k*r)/pow2(F1p_nk))*( (P_L(kargs[0])/pow2(F1kmp_nk))*p22 + 3.*(P_L(k)/pow2(F1_nk))*p13 );
```

Comment:
@billwright93 where does the "3" come from in the `3.*(P_L(k)/pow2(F1_nk))*p13` term? (I'm probably missing something trivial but the correspondence with the notes is not obvious still to me)

Reply:
I've forgotten where this comes from -- will check and get back to you.

Reply:
So at a glance the code expression doesn't match Eq.(2.25) from Ben's paper (1606.02520) which I've based my notes on. Hopefully @nebblu can explain what's happening here.

Reply:
There is a typo in Eq.2.25 of 1606.02520. In fact, it is inconsistent with 2.20 of 1606.02520 (which is correct).

Reply:
Ok, I've updated the Overleaf notes to correct the typo.